tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1400,29 +1371,38 @@ def test_continuous_writes_during_kpods_down(self):
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
                                  f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 6: Verified status for In-flight WRITEs while %s (K) pods "
+        LOGGER.info("Step 3: Verified status for In-flight WRITEs while %s (K) data pods "
                     "going down.", self.kvalue)
 
-        LOGGER.info("Step 7: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-35776-1'
-        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, nsamples=2,
-                                                    nclients=2)
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("STEP 4: Create IAM user and perform WRITEs-READs-Verify-DELETEs with "
+                        "variable object sizes on degraded cluster")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.test_prefix = 'test-35776-1'
+            self.s3_clean.update(users)
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix,
+                                                        nsamples=2, nclients=2)
+        else:
```

Comment:
same as above

Reply:
This is last step where we are performing IOs. as we discussed yesterday. It is not about READs WRITEs or DELETEs.